src/mutation_data.cpp

```diff
@@ -74,6 +74,54 @@ void load_mutation_category(JsonObject &jsobj)
     mutation_category_traits[id] = new_category;
 }
 
+static mut_attack load_mutation_attack( JsonObject &jo )
+{
+    mut_attack ret;
+    jo.read( "attack_text_u", ret.attack_text_u );
```

Comment:
Not required to merge but `assign` might be preferable as it offers easy extension to `strict` parsing of the JSON